Blueprints/BlueprintDefinitions/vs2022/CustomRuntimeFunction/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -44,7 +44,9 @@ Deploy function to AWS Lambda
 
 ## Arm64
 
-If you want to run your Lambda on an Arm64 processor, all you need is to do is add `"function-architecture": "arm64"` to the `aws-lambda-tools-defaults.json` file. Then deploy as described above.
+Graviton2 processors offer the best performance and price for running Lambda functions.
```

Comment:
I feel like this statement is too absolute and sounds too much like marketing for a instruction README file. Graviton2 is great and I know it is often the best performance/price for running Lambda but I don't think we can say that is always the best.

What do you think of this?

`Arm64 support is provided by the AWS Graviton 2 processor. For many Lambda work loads Graviton 2 provides the best price performance.`

Reply:
Updated as requested.